Data/Sys/Load/GraphicMods/Super Smash Bros Brawl/metadata.json

```diff
@@ -0,0 +1,72 @@
+{
+	"meta":
+	{
+		"title": "Performance Fixes",
+		"author": "iwubcode"
+	},
+	"groups":
+	[
+		{
+			"name": "OffscreenHoop",
+			"targets": [
+				{
+					"type": "efb",
+					"texture_filename": "efb1_n000022_4x4_6"
+				}
+			]
+		},
+		{
+			"name": "GameFinished",
+			"targets": [
+				{
+					"type": "draw_started",
+					"texture_filename": "tex1_400x148_5449617efae6c800_2"
+				}
+			]
+		},
+		{
+			"name": "GameFinishedNextScreen",
+			"targets": [
+				{
+					"comment": "C icon in complete for classic mode",
```

Comment:
Does this work for languages other than English?

Reply:
No, the comments in the file aren't translated.  It's not presented to the user in any way, it's just my way of adding a comment because json doesn't support it natively.

Reply:
I believe Jos is asking that if you set Brawl to a language other than English, would this work, given that the Clear! texture is likely localized.

Reply:
Oh apologies.  This would likely not work for other languages as they will have a different hash.  A separate target would be needed and I only have a NTSC copy unfortunately.